[2024-02-21T11:41:22.900+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-s3-mlflow.Start_Pipeline manual__2024-02-21T11:41:20.426402+00:00 [queued]>
[2024-02-21T11:41:22.907+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-s3-mlflow.Start_Pipeline manual__2024-02-21T11:41:20.426402+00:00 [queued]>
[2024-02-21T11:41:22.908+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T11:41:22.922+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Start_Pipeline> on 2024-02-21 11:41:20.426402+00:00
[2024-02-21T11:41:22.928+0000] {standard_task_runner.py:60} INFO - Started process 7668 to run task
[2024-02-21T11:41:22.931+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '***-s3-mlflow', 'Start_Pipeline', 'manual__2024-02-21T11:41:20.426402+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/***-aws-mlflow.py', '--cfg-path', '/tmp/tmpa35_n4dr']
[2024-02-21T11:41:22.934+0000] {standard_task_runner.py:88} INFO - Job 161: Subtask Start_Pipeline
[2024-02-21T11:41:22.953+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-21T11:41:22.989+0000] {task_command.py:423} INFO - Running <TaskInstance: airflow-s3-mlflow.Start_Pipeline manual__2024-02-21T11:41:20.426402+00:00 [running]> on host b14d6cdb7c29
[2024-02-21T11:41:23.060+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-s3-mlflow' AIRFLOW_CTX_TASK_ID='Start_Pipeline' AIRFLOW_CTX_EXECUTION_DATE='2024-02-21T11:41:20.426402+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-21T11:41:20.426402+00:00'
[2024-02-21T11:41:23.062+0000] {logging_mixin.py:188} INFO - Hello eClerx , MLOps process is started !!
[2024-02-21T11:41:23.063+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T11:41:23.072+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=***-s3-mlflow, task_id=Start_Pipeline, execution_date=20240221T114120, start_date=20240221T114122, end_date=20240221T114123
[2024-02-21T11:41:23.105+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T11:41:23.128+0000] {dagrun.py:822} ERROR - Failed to get task for ti <TaskInstance: airflow-s3-mlflow.save_model manual__2024-02-21T11:41:20.426402+00:00 [queued]>. Marking it as removed.
[2024-02-21T11:41:23.138+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
